setup.py

```diff
@@ -22,7 +23,7 @@
     author='ShopKeep.com Inc.',
     author_email='developers@shopkeep.com',
     url='https://github.com/shopkeep/shpkpr',
-    packages=['shpkpr', 'shpkpr.cli', 'shpkpr.commands', 'shpkpr.deployment', 'shpkpr.marathon', 'shpkpr.marathon_lb'],
+    packages=find_packages(),
```

Comment:
Using [find_packages](http://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages) here to avoid the ever growing list of manually specified packages. Note that our test modules aren't actually in real python packages so they don't need to be explicitly ignored.